source/localizable/tutorial/routes-and-templates.md

```diff
@@ -152,7 +152,7 @@ When we look at our about page at `http://localhost:4200/about`, we now have a w
 
 ![super rentals about page screenshot](../../images/routes-and-templates/ember-super-rentals-about.png)
 
-Now, we'll add a link to our contact page so we can navigate from back and forth between `about` and `contact`.
```

Comment:
This change isn't correct, we really do mean "add a link to", we just happen to use the `link-to` helper.